qt-models/divetripmodel.cpp

```diff
@@ -452,7 +452,7 @@ QVariant DiveTripModel::headerData(int section, Qt::Orientation orientation, int
 
 	switch (role) {
 	case Qt::TextAlignmentRole:
-		ret = dive_table_alignment(section);
+		ret = Qt::AlignLeft + Qt::AlignVCenter;
```

Comment:
Not wrong per se, but `ret = Qt::AlignLeft | Qt::AlignVCenter` would be more idiomatic.